packages/web-react/src/components/FileUploader/AttachmentImagePreview.tsx

```diff
@@ -1,20 +1,27 @@
 import React from 'react';
+import { FileQueueValueMetaType } from '../../types/fileUploader';
 import { useFileUploaderStyleProps } from './useFileUploaderStyleProps';
 import { IMAGE_DIMENSION } from './constants';
 
 type AttachmentImagePreviewProps = {
   label: string;
   imagePreview: string;
+  meta?: FileQueueValueMetaType;
```

Comment:
```suggestion
  meta?: FileMetadata;
```

Can we use more simple naming for the type, please? Consider using `FileMetadata` or `FileMeta` os something similar as we are using just `File` for the file type.